internal/config/path_test.go

```diff
@@ -0,0 +1,40 @@
+package config
+
+import (
+	"errors"
+	"testing"
+)
+
+//nolint:tparallel // t.Setenv forbits running tests in parallel
+func TestExpandPathWithoutTilde(t *testing.T) {
+	t.Run("rejects a path with a tildle", func(t *testing.T) {
+		t.Parallel()
+
+		path := "~/.config/gh-not"
+
+		got, err := ExpandPathWithoutTilde(path)
+		if !errors.Is(err, errTildeUsage) {
+			t.Errorf("want %v, got %v", errTildeUsage, err)
+		}
+
+		if got != "" {
+			t.Errorf("want empty string, got %q", got)
+		}
+	})
+
+	t.Run("expands environment variables", func(t *testing.T) {
+		t.Setenv("HOME", "/home/user")
+
+		path := "$HOME/.config/gh-not"
```

Comment:
Please add another test about using XDG_CONFIG_DIR

Reply:
Good idea!

Reply:
@ccoVeille, added in the latest commits. Have a look if you have time, I'll merge / release a new version later this week.